returnn/datasets/postprocessing.py

```diff
@@ -155,7 +165,10 @@ def __init__(
             self._out_tensor_dict_template = self._in_tensor_dict_template.copy_template()
             self.labels = self._dataset.labels.copy()
         # update only after _out_tensor_dict_template has been created from _in_tensor_dict_template
-        self._in_tensor_dict_template.update({"seq_tag": {"dims": (), "dtype": "string"}}, auto_convert=True)
+        self._in_tensor_dict_template.update(
+            {"complete_frac": {"dims": (), "dtype": "float64"}, "seq_tag": {"dims": (), "dtype": "string"}},
```

Comment:
Why float64 for complete_frac? We usually have float32 everywhere else (by default). I don't see why you need such high accuracy for the complete_frac.

Reply:
fixed